CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 ### Improvements 🛠
 
+* adds a new `error_mitigation` parameter on IonQ job submission so users can
+  configure strategies
+* adds a new `aggregation` parameter on IonQ results to allow getting results
+  aggregated under the two different methods described in
+  <https://arxiv.org/pdf/2301.07233.pdf>
```

Comment:
seems like your name got removed from the contributor list below, please feel free to add it back. also just updating this to follow our changelog conventions:
```suggestion
* adds a new `error_mitigation` parameter on IonQ job submission so users can
  configure strategies
  [(#75)](https://github.com/PennyLaneAI/PennyLane-IonQ/pull/75)

* adds a new `aggregation` parameter on IonQ results to allow getting results
  aggregated under the two different methods described in
  <https://arxiv.org/pdf/2301.07233.pdf>
  [(#75)](https://github.com/PennyLaneAI/PennyLane-IonQ/pull/75)
```